index.src.html

```diff
@@ -656,6 +656,31 @@ <h2 id="integration-with-html">Integration with HTML</h2>
 </section>
 
 <section>
+  <h2 id="integration-with-css">Integration with CSS</h2>
+
+  The CSS Standard does not specify how it fetches resources referenced from
+  stylesheets. However, implementations use the URL a given stylesheet was
```

Comment:
> we use "the document's URL" as referrer, without specifying this further.

No.  You use "the document's URL" with the "URL" bit being a link to the explicit definition of that term in the spec that defines it, and which can then be used to figure out what that URL actually is in any given circumstance.

> Would saying something like "the stylesheet's URL" address your concerns?

Not without a clear definition, either inline or via link to the relevant spec that does define it.